packages/vscode-extension/src/debug/teamsfxDebugProvider.ts

```diff
@@ -166,18 +169,24 @@ export class TeamsfxDebugProvider implements vscode.DebugConfigurationProvider {
       }
       debugConfiguration.url = result;
 
-      VsCodeLogInstance.info(sideloadingDisplayMessages.title(debugConfiguration.teamsfxHub!));
-      VsCodeLogInstance.outputChannel.appendLine("");
-      VsCodeLogInstance.outputChannel.appendLine(
-        sideloadingDisplayMessages.sideloadingUrlMessage(
-          debugConfiguration.teamsfxHub!,
-          debugConfiguration.url
-        )
-      );
-      if (isLocal) {
+      // NOTE: handle the case that msedge/chrome will be resolved twice
+      if (!debugConfiguration.resolved) {
```

Comment:
Better naming to `teamsfxResolved` to avoid potential conflict.

Reply:
Updated.